assignment.js

```diff
@@ -31,7 +31,11 @@ Use the following test cases to confirm your program meets the success criteria
 console.log('Problem 1:')
 
 // Add your code below this line
+const baseNumber = 2;
 
+for (i = 0; i <= 12; i++) {
+    console.log(baseNumber * i)
+}
 // Add your code above this line
 
```

Comment:
@OcielC Good job here implementing the for loop to complete the task. One suggestion for the conditionals, loops, and functions the standard is indentation by 2 spaces. Currently, you have 4 spaces so I wanted you to be aware of that.
Take a look at this alternate approach [here](https://github.com/jsr20190826/hw-01-js-basics-control-flow-loops-solution/blob/a810fe3f581584e5015519a3508b078d38c4c8c9/assignment.js#L34)